### DISCUSSION TITLE
Is it possible to define charset/collation for annotated columns in an entity ?

### DISCUSSION THREAD

**Discussion by ardabeyazoglu** [cycle/orm#370] [Q&A] [answered]
It would be nice to be able to define charsets for columns like this:

#[Entity]
class User
{
    #[Column(type: 'primary')]
    private int $id;

    #[Column(type: 'string', name: 'username', charset: 'latin1', collation: 'latin1_bin')]
    private string $email;
}

**Comment by roxblnfk** [comment, marked as answer]
I've created feature request #371
